### ISSUE TITLE
check sy subrc: false positive, also ok if dbcnt is checked

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1429]
its also okay if `DBCNT` is checked after a database operation, is DBCNT set for SELECT?

```abap
    UPDATE zabaplint_pack
      SET json = iv_json
      WHERE devclass = iv_devclass.
    ASSERT sy-dbcnt = 1.
```

**Comment by larshp** [comment]
done, closing
